lib/ethpy/ethpy/base/abi/load_abis.py

```diff
@@ -20,21 +31,27 @@ def load_all_abis(abi_folder: str) -> dict:
         A dictionary with keys for each abi filename and value is the "abi" field of the JSON decoded file
     """
     abis = {}
+    bytecodes = {}
     abi_files = _collect_files(abi_folder)
     loaded = []
     for abi_file in abi_files:
         file_name = os.path.splitext(os.path.basename(abi_file))[0]
         try:
-            abi_data = load_abi_from_file(abi_file)
+            abi_data = load_abi_from_file(abi_file, return_bytecode=return_bytecode)
+            if return_bytecode:
+                (abi_data, bytecode_data) = abi_data
+                bytecodes[file_name] = bytecode_data
             abis[file_name] = abi_data
             loaded.append(abi_file)
         except AssertionError as err:
             logging.debug("JSON file %s did not contain an ABI.\nError: %s", abi_file, err)
     logging.debug("Loaded ABI files %s", str(loaded))
+    if return_bytecode:
+        return abis, bytecodes
     return abis
 
 
-def load_abi_from_file(file_name: str) -> dict:
+def load_abi_from_file(file_name: str, return_bytecode: bool = False) -> dict | tuple[dict, dict]:
```

Comment:
shouldn't you have the `@overload` decorator for this func too?

Reply:
Doesn't look like it: https://adamj.eu/tech/2021/05/29/python-type-hints-how-to-use-overload/